src/requireCssModule.js

```diff
@@ -18,19 +18,57 @@ import type {
   StyleModuleMapType
 } from './types';
 
-const getTokens = (runner, cssSourceFilePath: string, filetypes): StyleModuleMapType => {
+type FileTypeOptions = {|
+  syntax: string,
+  plugins: Array<string>
```

Comment:
Please use `$ReadOnlyArray` and covariant props, i.e.  `{|+syntax: string, plugins: $ReadOnlyArray<string>|}`